mapie/_compatibility.py

```diff
@@ -26,8 +26,31 @@ def np_quantile_version_above_122(
     return np.quantile(a, q, method=method, **kwargs)  # type: ignore
 
 
+def np_nanquantile_version_below_122(
+    a: ArrayLike,
+    q: ArrayLike,
+    method: str = "linear",
+    **kwargs: Any
+) -> NDArray:
+    """Wrapper of np.quantile function for numpy version < 1.22."""
+    return np.nanquantile(a, q, interpolation=method, **kwargs)  # type: ignore
+
+
+def np_nanquantile_version_above_122(
+    a: ArrayLike,
+    q: ArrayLike,
+    method: str = "linear",
+    **kwargs: Any
+) -> NDArray:
+    """Wrapper of np.quantile function for numpy version >= 1.22."""
+    return np.nanquantile(a, q, method=method, **kwargs)  # type: ignore
+
+
 numpy_version = parse_version(np.__version__)
 if numpy_version < parse_version("1.22"):
     np_quantile = np_quantile_version_below_122
+    np_nanquantile = np_nanquantile_version_below_122
```

Comment:
nanquantile ignores nan. It can be slightly slower
quantile is not used in regression nor in time series regression, only in classification
and the opposite for nanquantile
Should test the performance between quantile and nanquantile before uniformization

Reply:
In the current master branch, `np_quantile` is used in `MapieRegressor`.